src/manifest.c

```diff
@@ -661,6 +692,11 @@ struct manifest *load_manifest(int current, int version, struct file *file, stru
 	manifest = manifest_from_file(version, file->filename, header_only);
 
 	if (manifest == NULL) {
+		if (retried == false) {
+			remove_manifest_files(file->filename, version);
+			retried = true;
+			goto retry_load;
+		}
```

Comment:
What if the bundle manifest hash check fails in `verify_bundle_hash`? Should that also retry?